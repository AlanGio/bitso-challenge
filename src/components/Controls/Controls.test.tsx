import React from "react";
import { render } from "@testing-library/react";
import { Controls } from "./index";

describe("<Controls />", () => {
  it("should render Controls component", () => {
    const { container } = render(
      <Controls
        columns={4}
        onChangeColumns={() => {}}
        onChangeRows={() => {}}
        rows={4}
        islands={2}
      />
    );

    expect(container).toMatchSnapshot();
  });
});
