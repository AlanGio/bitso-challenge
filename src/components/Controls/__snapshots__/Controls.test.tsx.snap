// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Controls /> should render Controls component 1`] = `
<div>
  <div
    class="component-controls"
  >
    <div
      class="controls-container container-fluid"
    >
      <form
        class=""
      >
        <div
          class="row"
        >
          <div
            class="col-sm-5 col-12"
          >
            <div
              class="form-group"
            >
              <label
                class="form-label"
                for="columnsRange"
              >
                Columns 
                4
              </label>
              <input
                class="form-control-range"
                id="columnsRange"
                max="50"
                min="2"
                type="range"
                value="4"
              />
            </div>
          </div>
          <div
            class="col-sm-5 col-10"
          >
            <div
              class="form-group"
            >
              <label
                class="form-label"
                for="rowsRange"
              >
                Rows 
                4
              </label>
              <input
                class="form-control-range"
                id="rowsRange"
                max="50"
                min="2"
                type="range"
                value="4"
              />
            </div>
          </div>
          <div
            class="col-sm-2 col-2"
          >
            Islands: 
            2
          </div>
        </div>
      </form>
    </div>
  </div>
</div>
`;
